doc/introduction/configuration.rst

```diff
@@ -36,20 +36,61 @@ and has the following format:
     port = 443
 
 Configuration options
-*********************
+---------------------
 
 **authentication_token (str)** (*required*)
-    The authentication token to use when connecting to the API. Will be sent with every request in
-    the header. Corresponding environment variable: ``SF_API_AUTHENTICATION_TOKEN``
+    API token for authentication to the Xanadu Cloud platform. This is required
+    for submitting remote jobs using :class:`~.StarshipEngine`. Corresponding
+    environment variable: ``SF_API_AUTHENTICATION_TOKEN``
 
 **hostname (str)** (*optional*)
-    The hostname of the server to connect to. Defaults to ``localhost``. Must be one of the allowed
-    hosts. Corresponding environment variable: ``SF_API_HOSTNAME``
+    The hostname of the server to connect to. Defaults to ``localhost``. Must
+    be one of the allowed hosts. Corresponding environment variable:
+    ``SF_API_HOSTNAME``
 
 **use_ssl (bool)** (*optional*)
     Whether to use SSL or not when connecting to the API. True or False.
     Corresponding environment variable: ``SF_API_USE_SSL``
 
 **port (int)** (*optional*)
     The port to be used when connecting to the remote service.
-    Corresponding environment variable: ``SF_API_PORT``
\ No newline at end of file
+    Corresponding environment variable: ``SF_API_PORT``
+
+Store your account
+------------------
+
+Using the :func:`~.store_account` function, a configuration file containing your Xanadu Cloud credentials
+can be created easily. By default, this configuration file is saved *globally*, and will be used every time
+a remote job is submitted.
+
+In these examples ``"MyToken"`` contains the user-specific authentication token.
+
+.. code-block:: python
+
+    import strawberryfields as sf
+    sf.store_account("MyToken")
+
+.. note::
+
+    By default, the account is stored in a ``config.toml`` file located in the
+    *Strawberry Fields configuration directory*.
+
+.. warning::
+    Typically, a user should only ever have to execute this code snippet once, when
+    initially configurating their system to connect to the Xanadu Cloud platform.
+    It is advised to call ``store_account`` **separately** from any other
+    Python code, such that the authentication token is not shared or committed
+    accidentally.
```

Comment:
Since this is user facing, better to not call the user using third person :)

```suggestion
.. warning::

    The ``store_account`` function only needs to be executed once, when
    initially configuring your system to connect to the Xanadu cloud platform.
    
    Take care not to share or publicly commit your authentication token, as it provides
    full access to your account.
```